tests/publiccloud/create_aistack_env.pm

```diff
@@ -188,6 +194,25 @@ sub install_aistack_chart {
     }    # pod loop
```

Comment:
In this point, having you detected eventual errors inpod/logs, is better to soon report and react here. Therefore I suggest to **move here** your checks performed below at lines _from 211 to 218_.

Moreover, also cosider if needed to exit on error detected in of log_pod: up to you, besed on system requirements.